src/redux/apis/auth_apis.tsx

```diff
@@ -8,8 +8,11 @@ export const registerAccountAPI = ({
   password: string;
 }) =>
   wretch('/api/auth/register')
+    .errorType('json')
     .post({
       email,
       password,
     })
+    .error(400, (error) => error.message)
```

Comment:
I could make the error codes generic if you want like always 400

Reply:
Ya, up to you